Makefile.test

```diff
@@ -31,32 +31,36 @@ localnet_init:
 	/bin/bash ./genlocalnetconfig.sh
 	/bin/bash ./genlocalnetappbins.sh
 
+localnet_init_latest_version:
+	/bin/bash ./genlocalnetconfig.sh
+	/bin/bash ./genlocalnetappbins.sh "" "" "" "true"
```

Comment:
Please add the latest version as a parameter (instead of just "true") and put it as a constant in this file.

Reply:
Done